Tests/PackageDescriptionTests/VersionTests.swift

```diff
@@ -531,7 +531,7 @@ class VersionTests: XCTestCase {
         
     }
     
-    // A semantic version string is always longer than 1 character, so the only way to test these 2 initialisers is by calling them directly.
+    // A semantic version string is always longer than 1 character, so the only way to test these 2 initializers is by calling them directly.
```

Comment:
Technically not a misspelling, but yes, American spelling is the standard for code.  :)